### PR TITLE
rearrangement + app scripts + testing locally

### PR DESCRIPTION
**Pull request by eancelet**
These are suggestions. We can discuss about it.
Don't hesitate to correct the English (and thanks) ;)

I need to add the section on testing locally.

We need to decide where to put the github repository and update the tutorial accordingly.

### CODE REVIEW

File: topics/dev/tutorials/interactive-tools/tutorial.md [hunk 1]
```diff
@@ -566,7 +593,7 @@ Have a look in the web interface of your Galaxy instance. You should find the ne
 > The [Interactive Tools admin tutorial](https://training.galaxyproject.org/training-material/topics/admin/tutorials/interactive-tools/tutorial.html)
 > demonstrates how this can be acheived by adding our tool XML to the "local tools" section of the Ansible Playbook. However, for our GxIT to show up in the correct tool panel we'll need to add an extra config file: `local_tool_conf.xml`.
 >
-> {% raw  %}
+>
```
Comment (neoformit) [inline topics/dev/tutorials/interactive-tools/tutorial.md]: The `{% raw %}` tag seems to be necessary to escape double braces `{{ }}` in the Liquid parser - if you look at the rendered webpage you'll see that `tool_path="{{ galaxy_local_tools_dir }}"` becomes `tool_path=""` without this tag!

File: topics/dev/tutorials/interactive-tools/tutorial.md [hunk 2]
```diff
@@ -472,6 +489,16 @@ our new Docker container as a Galaxy tool.
 >    >         Some help is always of interest ;)
 >    >
 >    >     ]]></help>
+>    >     <citations>
+>    >        <citation type="bibtex">
+>    >        @misc{
+>    >             author       = {Lain Pavot - lain.pavot@inrae.fr },
+>    >             title        = {{first-gxit -  A tool to visualise tsv/csv files }},
```
Comment (neoformit) [inline topics/dev/tutorials/interactive-tools/tutorial.md:498]: These double braces need to be escaped with a `{% raw %}` tag - see next comment

**Review by neoformit — CHANGES_REQUESTED** [review]
Hey Estelle, thanks for this! It all looks good, but there's one diff that I've requested changes on before merge as I think it breaks the rendering engine (see my comment about `{% raw %}` tag). I made some notes on things to polish as we go along but it's looking good for now.

File: topics/dev/tutorials/interactive-tools/tutorial.md (hunk 1 above)
Comment (neoformit) [reply topics/dev/tutorials/interactive-tools/tutorial.md]: You can see this rendered [on our dev GTN server](http://gtn.neoformit.com/training-material/topics/dev/tutorials/interactive-tools/tutorial.html#tip-install-a-gxit-with-an-ansible-playbook) under "Install a GxIT with an Ansible playbook".

Comment (eancelet) [reply topics/dev/tutorials/interactive-tools/tutorial.md]: Sorry, I didn't know about that and should have asked before I got rid of it.

File: topics/dev/tutorials/interactive-tools/tutorial.md (hunk 2 above)
Comment (eancelet) [reply topics/dev/tutorials/interactive-tools/tutorial.md:498]: Thanks for the tip. Raw tag added

File: topics/dev/tutorials/interactive-tools/tutorial.md (hunk 1 above)
Comment (eancelet) [reply topics/dev/tutorials/interactive-tools/tutorial.md]: Rectified

**Comment by eancelet** [comment]
I have also added the section about testing locally.
I found it easier to add this new commit to this pull request than making another pull request.

**Comment by neoformit** [comment]
Looks great, thanks for this :)

**Review by neoformit — APPROVED** [review]

**Pull request merged**